docs/reference/generated/kube-proxy.md

```diff
@@ -65,7 +65,7 @@ SupportIPVSProxyMode=true|false (ALPHA - default=false)
 TaintBasedEvictions=true|false (ALPHA - default=false)
 TaintNodesByCondition=true|false (ALPHA - default=false)
       --google-json-key string                       The Google Cloud Platform Service Account JSON Key to use for authentication.
-      --healthz-bind-address ip                      The IP address and port for the health check server to serve on (set to 0.0.0.0 for all interfaces) (default 0.0.0.0:10256)
+      --healthz-bind-address ip                      The IP address and port for the health check server to serve on (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 0.0.0.0:10256)
```

Comment:
should be fixed here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L131